specs/management.yml

```diff
@@ -15962,6 +15962,8 @@ definitions:
         items:
           type: string
       encryptionRequired:
+        description: Describes whether connections must support end-to-end encryption
+          on both sides of the connection. Read-only property, set at create.
```

Comment:
this 'is' the ServiceCreate. feels like this snippet is contradictory here: "Read-only property, set at create"?

Reply:
Yeap, find and replace got me. Fixed in a second.